src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingUI.js

```diff
@@ -18,26 +17,66 @@ export const BenchmarkingLayout = ({
     showBamQCLink = false,
     bamQCHash = '', // TODO: Other datasets will have qcs on same page accessible by anchor hash
     children,
+    setShowInformation = null,
     callout = null,
+    showInformation = true,
```

Comment:
Is there a reason the useState for these variables is not set in `<BenchmarkingLayout/>`? It's awfully redundant to have to define that in every single data component, and it seems like you're just passing those into this component (the only place where they are actually used), anyway.

Reply:
Sorry yes, definitely redundant. I was previously passing the state down to HashBasedTabController to adjust table height and did not update. Thanks for pointing this out